source/core/Synchronization.cpp

```diff
@@ -1168,16 +1165,19 @@ DEFINE_VIREO_BEGIN(Synchronization)
     // Dynamic, refnum-based queues
     DEFINE_VIREO_TYPE(QueueRefNum, "refnum(Queue)")
     DEFINE_VIREO_FUNCTION_CUSTOM(ObtainQueue, QueueRef_Obtain,
-        "p(o(QueueRefNum queue)i(Int32 maxsize)i(String name)i(Boolean create)o(Boolean created)io(ErrorCluster err))")
-    DEFINE_VIREO_FUNCTION_CUSTOM(ReleaseQueue, QueueRef_Release, "p(i(QueueRefNum queue)o(String name)o(Array remainingElems)io(ErrorCluster err))")
+        "p(i(StaticTypeExplicitData)o(QueueRefNum queue)i(Int32 maxsize)i(String name)i(Boolean create)o(Boolean created)io(ErrorCluster err))")
+    DEFINE_VIREO_FUNCTION_CUSTOM(ReleaseQueue, QueueRef_Release, "p(i(StaticTypeExplicitData)i(QueueRefNum queue)o(String name)"
+                                 "o(Array remainingElems)io(ErrorCluster err))")
     DEFINE_VIREO_FUNCTION_CUSTOM(Enqueue, QueueRef_Enqueue,
-        "p(io(QueueRefNum queue)i(* element)i(Int32 timeOut)o(Boolean timedOut)io(ErrorCluster err))")
+        "p(i(StaticTypeExplicitData)io(QueueRefNum queue)i(* element)i(Int32 timeOut)o(Boolean timedOut)io(ErrorCluster err))")
```

Comment:
I'll do so, but keep in mind these strings are actually embedded in the exe/vireo.js file and parsed at launch, so I'm not sure it's a good trade-off to do that for a very minor increase in readability.  It would be nice if these prim-defining string could be stored in an out-board file and discarded after parsing.